tools/export/iar.py

```diff
@@ -66,6 +66,8 @@ def generate(self):
             }
 
         project_data['tool_specific'] = {}
+        project_data['tool_specific'].setdefault("iar", {})
+        project_data['tool_specific']['iar'].update(self.progen_flags)
```

Comment:
flags should be in misc, I got a local patch, IAR builds! will send another PR on top of this one.